config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -50,6 +50,15 @@ presets:
     value: 1000
   - name: LOGROTATE_MAX_SIZE
     value: "5G"
+  # Ensure good enough architecture for master machines.
+  - name: MASTER_MIN_CPU_ARCHITECTURE
+    value: "Intel Broadwell"
+  # Increase delete collection parallelism.
+  - name: TEST_CLUSTER_DELETE_COLLECTION_WORKERS
+    value: --delete-collection-workers=16
+  # Dump full systemd journal on master and nodes.
```

Comment:
No sure if we want this one actually - this is about main nodes, not hollow nodes.

Reply:
I added it mostly because of kubemark master. Gathering logs from kubemark nodes fails anyway and it's skipped. Logs from ci-job:
```
I0729 13:26:09.655] Detecting nodes in the cluster
W0729 13:26:19.782] scp: /var/log/glbc.log*: No such file or directory
W0729 13:26:19.782] scp: /var/log/cluster-autoscaler.log*: No such file or directory
W0729 13:26:19.865] scp: /var/log/fluentd.log*: No such file or directory
W0729 13:26:19.866] scp: /var/log/kubelet.cov*: No such file or directory
W0729 13:26:19.867] scp: /var/log/startupscript.log*: No such file or directory
W0729 13:26:19.867] ERROR: (gcloud.compute.scp) [/usr/bin/scp] exited with return code [1].
I0729 13:26:20.013] Skipping dumping of node logs
```
I ran a test locally and I didn't observe any additional errors.

Reply:
We have logs from hollow-nodes, e.g.:
https://gcsweb.k8s.io/gcs/kubernetes-jenkins/logs/ci-kubernetes-kubemark-100-gce/1155823300224487425/artifacts/kubemark-100-minion-group-0654/

My question is: why do we need it? Maybe what we want it to remove form the gce instead?

Reply:
That's a good question...
IIRC I never needed to use systemd logs. It is mostly a copy of kubelet.log + docker.log.

Reply:
OK anyway - I think it's not crucial, but let's keep it for now. Let's unify and then we will be cleaning up.